lesson-20/example-github-login/main.py

```diff
@@ -47,10 +36,7 @@ def github_callback():
                                authorization_response=request.url)
 
     response = make_response(redirect(url_for('profile')))  # redirect to the profile page
-    if local:
-        response.set_cookie("oauth_token", json.dumps(token), httponly=True)
-    else:
-        response.set_cookie("oauth_token", json.dumps(token), httponly=True, samesite='Strict')
+    response.set_cookie("oauth_token", json.dumps(token), httponly=True)
```

Comment:
@Raffson have you checked if httponly=True works on localhost? This might be also problematic...

Reply:
Nevermind, I tested it and it works.